templates/lune/data.json

```diff
@@ -0,0 +1,47 @@
+{
+  "lune": {
+    "head-tag": {
+      "extensions": [
+        "amp-sidebar"
+      ],
+      "font-stylesheets": [
+        "https://fonts.googleapis.com/css?family=Lato:400,700,900"
+      ]
+    },
+    "navbar": {
+      "name": "AMP Lune template",
+      "sidebar-id": "header-sidebar",
+      "sidebaronly": true,
+      "href": "lune.amp.html",
+      "logo": {
+         "name": "LUNE",
+         "classes": "h3 xs-hide"
+      },
+      "fixeditems": [
+        {
+          "link": {
+            "url": "mailto:contact@lune.com",
+            "text": "—contact@lune.com"
+          }
+        }
+      ],
+      "nav": [
+        {
+          "link": {"url": "lune.amp.html", "text": "<span>H</span><span>o</span><span>m</span><span>e</span>", "aria-label": "Home"}
+        },
+        {
+          "link": {"url": "product-blanc.amp.html", "text": "<span>B</span><span>l</span><span>a</span><span>n</span><span>c</span>", "aria-label": "Blanc"}
```

Comment:
I've acknowledged your comment about accessibility. We had a couple of options to get past this:

1. Add an `aria-label` on the first span and add `aria-hidden` on the others
2. Add an `aria-label` property to the link object, which is then applied to the anchor tag in sidebar.snip.html

We decided the second option was cleaner. I'm interested to see what you think about this approach.